.gitattributes

```diff
@@ -1 +1,3 @@
-zz_generated_*.go linguist-generated=true
+**/zz_generated*.go linguist-generated=true
+pkg/apis/** linguist-generated=true
```

Comment:
This one I'd suggest to leave out since this is where the go structures are defined, by the developers.

```suggestion
```

Reply:
Good catch. There are only `zz_generated.deepcopy.go` which will get caught by the `zz_generated_*.go`.